docs/style-spec/_generate/index.html

```diff
@@ -529,8 +529,8 @@ <h3 class='space-bottom1'><a href='#sources-canvas' title='link to canvas'>canva
               <td>basic functionality</td>
               <td>&gt;= 0.32.0</td>
               <td>Not supported</td>
-              <td>Not supported</td>
-              <td>Not supported</td>
+              <td>&gt;= 3.6.0</td>
```

Comment:
This is the table for `canvas` source support. You mean to change the `image` source support table, right?